src/components/facets/FacetRow.tsx

```diff
@@ -9,48 +10,96 @@ export interface IFacetRowProps extends React.ClassAttributes<FacetRow> {
     onToggleFacet: (facetRow: IFacet) => void;
     isChecked: boolean;
     maxTooltipLabelLength?: number;
+    enabledExclude?: boolean;

```

Comment:
Same here, I think renaming `enabledExclude` for `excludeEnabled`, `exclusionsEnabled` or `enableExclusions` would make more sense